test/e2e/tsdx-build-default.test.ts

```diff
@@ -41,6 +41,13 @@ describe('tsdx build :: zero-config defaults', () => {
     expect(output.code).toBe(0);
   });
 
+  it('should add shebang', async () => {
+    const output = execWithCache('node ../dist/index.js build');
```

Comment:
To test shebangs, you'd want to exec `. ./dist/index.js` so that the shebang gets used